### PR TITLE
Adpanel test

### PR DESCRIPTION
**Pull request by michaldzwigala**
Update adpanel tests.

### CODE REVIEW

File: src/app/common/components/notifications/notification/notification.component.html [hunk 1]
```diff
@@ -33,7 +34,7 @@
     >
       {{ notification.message }}
     </p>
-    <div
+    <!-- <div
```
Comment (adshares-tech) [inline src/app/common/components/notifications/notification/notification.component.html]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)

File: src/app/common/components/notifications/notification/notification.component.html [hunk 2]
```diff
@@ -4,12 +4,13 @@
     notification"
 >
   <div class="col-xs-2">
-    <img
+    <!-- TODO : wymaga posprzątania i przebudowy, pociągnąć logikę z serverem -->
+    <!-- <img
```
Comment (adshares-tech) [inline src/app/common/components/notifications/notification/notification.component.html]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)

File: src/app/common/components/notifications/notifications.component.html [hunk 3]
```diff
@@ -16,7 +16,8 @@
       class="notification-wrapper"
       *ngFor="let notification of notifications"
     >
-      <app-notification
+      <!-- TODO: to wymaga całkowitej przebuowy, już wiem jak dokoncze wieczorem -->
+      <!-- <app-notification
```
Comment (adshares-tech) [inline src/app/common/components/notifications/notifications.component.html]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)

File: src/app/settings/general-settings/preferences/preferences.component.html [hunk 4]
```diff
@@ -238,12 +202,12 @@
             Change password
           </button>
         </div>
-        <div
+        <!-- <div
```
Comment (adshares-tech) [inline src/app/settings/general-settings/preferences/preferences.component.html]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)

File: src/app/settings/general-settings/notification-settings/notification-settings.component.html [hunk 5]
```diff
@@ -57,6 +57,20 @@
...
+              ></mat-slide-toggle>
+            </div>
+            <div class="col-xs-2">
+              <mat-slide-toggle
+                color="primary"
+                aria-label="notification-email"
+                [checked]="notification.email === true"
+              ></mat-slide-toggle>
+            </div>
+            <!-- <div class="col-xs-4">
```
Comment (adshares-tech) [inline src/app/settings/general-settings/notification-settings/notification-settings.component.html]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)

**Comment by adshares-tech** [comment]
SonarQube analysis reported 14 issues
* ![MAJOR][MAJOR] 14 major

Watch the comments in this conversation to review them.

#### 9 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [forgotten-password.component.html#L111](https://github.com/adshares/adpanel/blob/1c667ac53cd13421867d06e80b1315a97901c614/src/app/auth/forgotten-password/forgotten-password.component.html#L111): Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)
1. ![MAJOR][MAJOR] [forgotten-password.component.html#L117](https://github.com/adshares/adpanel/blob/1c667ac53cd13421867d06e80b1315a97901c614/src/app/auth/forgotten-password/forgotten-password.component.html#L117): Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)
1. ![MAJOR][MAJOR] [forgotten-password.component.html#L122](https://github.com/adshares/adpanel/blob/1c667ac53cd13421867d06e80b1315a97901c614/src/app/auth/forgotten-password/forgotten-password.component.html#L122): Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)
1. ![MAJOR][MAJOR] [forgotten-password.component.html#L124](https://github.com/adshares/adpanel/blob/1c667ac53cd13421867d06e80b1315a97901c614/src/app/auth/forgotten-password/forgotten-password.component.html#L124): Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)
1. ![MAJOR][MAJOR] [forgotten-password.component.html#L130](https://github.com/adshares/adpanel/blob/1c667ac53cd13421867d06e80b1315a97901c614/src/app/auth/forgotten-password/forgotten-password.component.html#L130): Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)
1. ![MAJOR][MAJOR] [forgotten-password.component.html#L131](https://github.com/adshares/adpanel/blob/1c667ac53cd13421867d06e80b1315a97901c614/src/app/auth/forgotten-password/forgotten-password.component.html#L131): Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)
1. ![MAJOR][MAJOR] [forgotten-password.component.html#L134](https://github.com/adshares/adpanel/blob/1c667ac53cd13421867d06e80b1315a97901c614/src/app/auth/forgotten-password/forgotten-password.component.html#L134): Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)
1. ![MAJOR][MAJOR] [forgotten-password.component.html#L135](https://github.com/adshares/adpanel/blob/1c667ac53cd13421867d06e80b1315a97901c614/src/app/auth/forgotten-password/forgotten-password.component.html#L135): Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)
1. ![MAJOR][MAJOR] [forgotten-password.component.html#L136](https://github.com/adshares/adpanel/blob/1c667ac53cd13421867d06e80b1315a97901c614/src/app/auth/forgotten-password/forgotten-password.component.html#L136): Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by michaldzwigala** [comment]
update

**Pull request closed without merging**